Gopkg.toml

```diff
@@ -77,7 +77,7 @@ ignored = [
 
 [[constraint]]
   name = "github.com/gravitational/satellite"
-  version = "=7.0.3"
+  version = "7.0.5"
```

Comment:
Let's keep the leading `=` there. Otherwise dep might update the dependencies to a newer patch version unintentionally.